nexus/db-model/src/vpc_firewall_rule.rs

```diff
@@ -264,6 +265,20 @@ impl VpcFirewallRule {
     }
 }
 
+const MAX_FIREWALL_RULES: usize = 32;
```

Comment:
Is this an arbitrary number? Perhaps it'd be  good to have a comment that explains why we chose this limit?

Reply:
This number feels way too low. I would expect a value in the hundreds at least, especially since we don't have tags for grouping firewall rules.

Reply:
Yeah, definitely. Was going to ask about the number next. How about 512?